src/app/zone-ingresses/views/item/ServicesView.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <RouteView
+    v-slot="{ t }"
+    name="zone-ingress-services-view"
+  >
+    <AppView>
+      <template #title>
+        <h2>
+          <RouteTitle
+            :title="t('zone-ingresses.routes.item.navigation.zone-ingress-services-view')"
+            :render="true"
+          />
+        </h2>
+      </template>
+
+      <KCard class="mt-4">
+        <template #body>
+          <template
+            v-for="items in [
+              ...(props.data?.zoneIngress?.availableServices ? [props.data.zoneIngress.availableServices] : [])
+            ]"
+            :key="items"
+          >
+            <AppCollection
+              :empty-state-message="t('common.emptyState.message', { type: 'Services' })"
+              :headers="[
+                { label: 'Name', key: 'name' },
+                { label: 'Mesh', key: 'mesh' },
+                { label: 'Protocol', key: 'protocol' },
+                { label: 'No. Instances', key: 'instances' },
+                { label: 'Actions', key: 'actions', hideLabel: true },
+              ]"
+              :items="items"
+            >
+              <template #name="{ row: item }: {row: AvailableService}">
+                <template
+                  v-for="service in [
+                    findTag(item.tags, 'kuma.io/service') ?? t('common.collection.none')
```

Comment:
**Suggestion**: Since we need to process the availableServices data further, we should use a computed property and include the results of `findTag` as part of a new array.

Reply:
I removed `findTag` entirely, as I figured we could just access the property here